packages/hardware-trezor/package.json

```
@@ -63,6 +63,7 @@
     "@cardano-sdk/key-management": "workspace:~",
     "@cardano-sdk/tx-construction": "workspace:~",
     "@cardano-sdk/util": "workspace:~",
+    "lodash": "4.17.21",
```

How about more flexible versioning:
```suggestion
    "lodash": "^4.17.21",
```
We seem to be declaring dependencies like that everywhere else in SDK. Strict versions can easily lead to duplication of dependencies.